packages/field-base/src/input-mixin.js

```diff
@@ -52,14 +52,24 @@ export const InputMixin = dedupingMixin(
             observer: '_valueChanged',
             notify: true,
           },
+
+          /**
+           * Populated state of the input's value.
+           * @private
+           */
+          __inputValuePopulated: {
```

Comment:
Would it make sense to use something like `_hasInputValue` to make it clear it's a boolean property?
Also, consider making this property protected to make it clear that it's used outside this mixin.

Reply:
Good point! Updated.